README.md

```diff
@@ -1,59 +1,154 @@
 Awesome Go Books [![Build Status](https://github.com/dariubs/GoBooks/actions/workflows/main.yml/badge.svg?branch=master)](https://github.com/dariubs/GoBooks/actions/workflows/main.yml) [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
 ====
-* [Books](#books)
- * [Starter Books](#starter-books)
- * [Advanced Books](#advanced-books)
- * [Web Development](#web-development)
-* [Resources](#resources)
+- [Awesome Go Books  ](#awesome-go-books--)
+- [**Books**](#books)
+    - [**Starter Books**](#starter-books)
+        - [2018 - Learning Go *Free*](#2018---learning-go-free)
+        - [2018 - Get Programming with Go](#2018---get-programming-with-go)
+        - [2019 - API Foundations in Go](#2019---api-foundations-in-go)
```

Comment:
What's the best way to maintain this table of contents?

Should the CONTRIBUTING file recommend installing that VS Code plugin mentioned in the pull request?

Reply:
I would recommend that, I didn't check the license but I don't see any stopper there, if it becomes under payment we can just switch to another plugin 

With that you just CTRL+SHIFT+P and update
![image](https://github.com/dariubs/GoBooks/assets/8364469/74407acb-3e87-4c14-a8a7-6213021b57f3)